### PR TITLE
safhe avalo tarahi kardam

### PR DESCRIPTION
**Pull request by hesamde**
siavash taeed kon

### CODE REVIEW

File: lab/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="test2.js"> </script>
+</head>
+<body>
+    <button onclick="add(10,5)"> click me </button>
+</body>
+</html>
```
Comment (SiaExplains) [inline lab/index.html]: we should add an empty space at the end of all files in Github

File: lab/test2.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 function add(n1, n2) {
-  return n1 + n2;
-}
\ No newline at end of file
+  console.log(n1+n2); 
+}
```
Comment (SiaExplains) [inline lab/test2.js:3]: same space problem

**Review by SiaExplains — CHANGES_REQUESTED** [review]
please apply my suggestions

File: lab/index.html (hunk 1 above)
Comment (hesamde) [reply lab/index.html]: i didnt know about the convention

**Review by SiaExplains — APPROVED** [review]

**Pull request merged**